protostar/commands/test/test_command.py

```diff
@@ -155,6 +155,11 @@ def arguments(self):
                 type="bool",
                 description="Show gas estimation for each test case. Estimations might be inaccurate.",
             ),
+            ProtostarArgument(
+                name="compiled-contracts-dir",
```

Comment:
Shouldn't this one be stored in .protostar by default? Do you think that it is necessary to give user option to change the location?